### ISSUE TITLE
{Linked:Bug9210947}[Adaptive Cards> Previous/Next Button]: Narrator is announcing the previous and next button as greater/less <> than.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7657]
### Target Platforms

Other

### SDK Version

1.3

### Application Name

Adaptive cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25145.1011)
Browser: Edge Dev (105.0.1300.0)
Screen Reader: Narrator /JAWS
URL: https://adaptivecards.io/designer/

**Repro Steps:**
1. Turn on the Narrator through Ctrl+ windows+ Enter key.
2. Open the above URL in edge dev browser.
3. Add the provided JSON in Sample data editor and card payload editor.
4. Navigate to the previous and Next button through Tab key.
5. Observe the issue.

**Actual Result:**
Narrator is announcing the previous and next button as greater/less <> than.

**Expected Result:**
Narrator should announce the previous and next button properly so its convey the meaning as where less than sign present it should announce as 'Previous button' and for greater than sign it should announce as 'Next Button'.

**Observation:**
Same issue repro with JAWS.

**User Impact:**
User who relies on the screen reader will be impacted as it is not announcing the name of functional elements present in an appropriate way.

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

"**Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

https://user-images.githubusercontent.com/104483005/178416216-9da7d371-7f76-4c72-a957-e5a3218f665f.mp4

### Card JSON

```JSON
{
  "type": "AdaptiveCard",
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.0",
  "msTeams": { "width": "full" },
  "body": [
    {
      "type": "TextBlock",
      "text": "Great! Here are some of the things that Icebreaker can help you with. Feel free to give it a try!",
      "wrap": true
    },
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "verticalContentAlignment": "Center",
          "items": [
            {
              "type": "ActionSet",
              "actions": [
                {
                  "type": "Action.ToggleVisibility",
                  "title": "<",
                  "targetElements": [
                    "imageToToggle1",
                    "imageToToggle2",
                    "textToToogle1",
                    "textToToogle1-1",
                    "textToToggleWelcomeFooter",
                    "textToToogle2-1",
                    "textToToogleMatchFooter",
                    "textToToogle2"
                  ]
                }
              ]
            }
          ],
          "width": "stretch"
        },
        {
          "type": "Column",
          "width": "400px",
          "items": [
            {
              "type": "Image",
              "url": "${tour1Url}",
              "id": "imageToToggle1",
              "altText": "Tour Image one"
            },
            {
              "type": "Image",
              "url": "${tour2Url}",
              "id": "imageToToggle2",
              "isVisible": false,
              "altText": "Tour Image two"
            }
          ]
        },
        {
          "type": "Column",
          "verticalContentAlignment": "Center",
          "items": [
            {
              "type": "ActionSet",
              "actions": [
                {
                  "type": "Action.ToggleVisibility",
                  "title": ">",
                  "targetElements": [
                    "imageToToggle1",
                    "imageToToggle2",
                    "textToToogle1",
                    "textToToogle1-1",
                    "textToToggleWelcomeFooter",
                    "textToToogle2-1",
                    "textToToogleMatchFooter",
                    "textToToogle2"
                  ]
                }
              ]
            }
          ],
          "width": "stretch"
        }
      ]
    },
    {
      "type": "TextBlock",
      "text": "${tourWelcomeHeader}",
      "wrap": true,
      "size": "Large",
      "weight": "Bolder",
      "id": "textToToogle1"
    },
    {
      "type": "TextBlock",
      "text": "${tourWelcomebodyText}",
      "wrap": true,
      "id": "textToToogle1-1"
    },
    {
      "type": "TextBlock",
      "text": "${tourFooterText}",
      "wrap": true,
      "id": "textToToggleWelcomeFooter"
    },
    {
      "type": "TextBlock",
      "text": "${tourMatchHeader}",
      "wrap": true,
      "size": "Large",
      "weight": "Bolder",
      "id": "textToToogle2",
      "isVisible": false
    },
    {
      "type": "TextBlock",
      "text": "${tourMatchBody}",
      "wrap": true,
      "id": "textToToogle2-1",
      "isVisible": false
    },
    {
      "type": "TextBlock",
      "text": "${tourMatchFooter}",
      "wrap": true,
      "id": "textToToogleMatchFooter",
      "isVisible": false
    }
  ]
}
```

**Comment by anna-dingler** [comment]
Hello @vaishali1397! To resolve this issue, I recommend using an icon instead of "<" and ">" and assigning a tooltip value.

For instance, 
```
{
  "type": "ActionSet",
  "actions": [
    {
      "type": "Action.ToggleVisibility",
      "iconUrl": "icon_url",
      "tooltip": "Previous",
      "targetElements": [
        "imageToToggle1",
        "imageToToggle2",
        "textToToogle1",
        "textToToogle1-1",
        "textToToggleWelcomeFooter",
        "textToToogle2-1",
        "textToToogleMatchFooter",
        "textToToogle2"
      ]
    }
  ]
}
```

The screen reader would then properly announce "Previous Button". Let me know if you have any questions!

**Comment by anna-dingler** [comment]
Resolving this issue since using an icon would result in the correct behavior. Let me know if you need help with anything else!

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
